packages/repl/src/index.js

```diff
@@ -12,6 +12,7 @@ const {
 } = require("webassemblyjs/lib/interpreter/kernel/memory");
 const { decode } = require("@webassemblyjs/wasm-parser");
 const t = require("@webassemblyjs/ast");
+const typeCheck = require("@webassemblyjs/validation").stack;
```

Comment:
@xtuc I hope I didnt misunderstand things but to me it looked like no validation was running in the repl at all? I added it in a way that it is currently only run when using `assert_invalid` with `type mismatch` as expected error. As expected some case with `br` and `return` fails :grin: